### ISSUE TITLE
Last question. I look forward to your response.

### ISSUE THREAD

**Issue by JoinWorldMC** [lhoyer/HRDA#23]
@lhoyer

In the paper. Figure 2(b) includes "Reassemble".

What I mean by forward pass is whether or not Reassemble is performed only once.
That is, parts of the image should not be inferred multiple times.

In Case 1, does Reassemble work once?

**Comment by JoinWorldMC** [comment]
@lhoyer 

Also, am I wrong to think that HRDA is a resolution-based ensemble method?

HRDA is not strictly an ensemble method as there is only one model trained.

HRDA is not an ensemble because there is only one model, but it looks similar to Test Time Adaptation. Isn't it?

**Comment by lhoyer** [comment]
Duplicate of #21
